iree/hal/vulkan/vulkan_driver.cc

```diff
@@ -17,13 +17,17 @@
 #include <memory>
 
 #include "absl/container/inlined_vector.h"
+#include "absl/flags/flag.h"
 #include "iree/base/memory.h"
 #include "iree/base/status.h"
 #include "iree/base/tracing.h"
 #include "iree/hal/device_info.h"
 #include "iree/hal/vulkan/extensibility_util.h"
 #include "iree/hal/vulkan/status_util.h"
 
+ABSL_FLAG(bool, vulkan_enable_renderdoc, false,
```

Comment:
<meme> drop the 'enable', it's cleaner

(but srsly, for bool flags presence == enable)

Reply:
Done